invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html

```diff
@@ -113,17 +122,20 @@
 
 {% macro file_list_box(files, pid, is_preview, record) %}
 {%- set binary_sizes = not config.APP_RDM_DISPLAY_DECIMAL_FILE_SIZES %}
-<div class="">
-  <div class="ui accordion panel mb-10 {{record.ui.access_status.id}}" href="#collapsablePreview">
-    <div class="active title trigger panel-heading {{record.ui.access_status.id}}" tabindex="0">
-      {{ _("Files") }}
-      <small class="text-muted">{% if files %} ({{files|sum(attribute='size')|filesizeformat(binary=binary_sizes)}}){% endif %}</small>
-      <i class="angle right icon"></i>
-    </div>
-    <div class="active content pt-0">
+{%- set files_access = record.access.files if not is_media else record.access.media_files %}
```

Comment:
I think @anikachurilova changed this so probably align with her first if this change makes sense?

Reply:
I removed the line `set files_access` (see the commit https://github.com/inveniosoftware/invenio-app-rdm/commit/47194f416a2379b69675611d9dc2ce85df3dd1f9) because the var `files_access` is not used anywhere. If it's still needed, then the  `is_media` should be put back to the `file_list_box` params (and apply it to where it's used) as it is used there. 
Also, I see some changes to the UI, does it make sense to also apply them to the ` media_file_list_box`? Let me know if you need help with adding media files to the record to test it.